src/sql/workbench/contrib/executionPlan/browser/executionPlanView.ts

```diff
@@ -446,24 +446,31 @@ export class SavePlanFile extends Action {
 	public override async run(context: ExecutionPlanView): Promise<void> {
 		const workspaceFolders = context.workspaceContextService.getWorkspace().folders;
 		const defaultFileName = 'plan';
+		const fileExtension = 'sqlplan'; //TODO: Get this extension from provider
```

Comment:
It would be good to take the opportunity and replace all hardcoded instances of sqlplan in the code with a single constant defined somewhere - possibly src\sql\platform\connection\common\constants.ts